testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewUITests.m

```diff
@@ -35,6 +35,29 @@ - (instancetype)initWithInvocation:(NSInvocation*)invocation {
 }
 
 - (void)testPlatformView {
+  //[self checkGolden];
+  [[XCUIDevice sharedDevice] pressButton:XCUIDeviceButtonHome];
+  [self.application activate];
+  [self checkGolden];
+}
```

Comment:
It seems we are also testing the background and foreground thingy here. Maybe we should rename the test to be more specific about it? (Like what is the difference between this test and the test below)
Or if you didn't mean to test the background and foreground here, you can remove the logic.

Reply:
🤦‍♂ this part should be reverted, I started by experimenting on this test. I'll followup in a new PR